feature-libs/cart/import-export/components/export-entries/export-entries.service.ts

```diff
@@ -108,4 +129,15 @@ export class ExportEntriesService {
       })
     );
   }
+
+  downloadCsv(entries: string[][]) {
+    if (this.exportConfig.messageEnabled) this.displayExportMessage();
+
+    this.exportCsvService.downloadCsv(
+      this.exportCsvService.dataToCsv(entries),
+      this.exportConfig.fileName,
+      undefined,
```

Comment:
what do you think about allowing to pass some extension from config here? With default `.csv` - I know that it will be only one file format (it could be also considered), but the extension is contractual

Reply:
Because method is called `downloadCsv` maybe worth to consider is to remove extension from params?